openapi-module/temp/curl-commands.sh

```diff
@@ -0,0 +1,5 @@
+curl -X GET "http://localhost:3000/users?limit=557"
```

Comment:
I believe temp folder should not exist in the repo. All files there will be generated, no need to have them pre-stored.

Reply:
But when I removed temp, the test wouldn't be successful, I think we may keep temp but remove files under it

Reply:
Then you should create temp folder (outputDir variable) before the tests run. You may use that before function statement.